packages/core/src/lib/upload.spec.ts

```diff
@@ -34,18 +36,33 @@ vi.mock('fs/promises', async () => {
 });
 
 const outputDir = MEMFS_VOLUME;
-const reportPath = (path = outputDir, format: 'json' | 'md' = 'json') =>
-  join(path, 'report.' + format);
+const filename = reportFileName({ date: new Date().toISOString() });
```

Comment:
Again, the timestamps in the filename are unnecessary here, these test have full control over the state of "file system" because they use `memfs`.